website/blog/2018-01-10-1.10.0.md

```diff
@@ -0,0 +1,411 @@
+
+Snapshot tests are a wonder for a pretty printer. Adding new tests is dead easy, just create a file in the test folder with the code you want to format and run jest, voila! Anytime you change something, you can see how a bunch of examples would be printed differently, then it's up to you to decide if it's better or not. For a reviewer, it's also really nice, you can look at the before/after of all the things it changed. I've come to pay more attention to the snapshots than the actual implementation.
+
+[The playground](https://prettier.io/playground) is a really nice way to get to a repro or play with Prettier without having to install the development environment, being on the right branch... This has proven to be extremely valuable to get people to provide really good bug reports that are actionable.
+
+## Highlights
+
+### Introduce proper support for Vue Single File Components ([#3563]) by [@vjeux]
+
+There's a lot of demand for Vue SFC ([#2097]). We've introduced partial support for them: All the HTML is printed as is, but now the `<script>` and `<style>` tags are printed using Prettier.
+
+To use it, just run `prettier` on your `*.vue` files!
+
+### API
+
+#### Create the Prettier Plugin API ([#3536]) by [@azz]
+
+As Prettier for JavaScript has become stable, we've recently had contributors wanting to add new languages to Prettier, notably we had pull requests to add Python support and PHP support. We want to keep the core `prettier` package portable and maintainable, but we also want to give people the opportunity to run Prettier on more languages. To that end we've introduced a [plugin API]! Prettier plugins can contribute parsers and/or printers to the Prettier formatter. They are treated as first-class citizens and can even contribute to the embedding support (e.g. formatting your language inside Markdown).
+
+Using plugins is as simple as installing them via `npm`/`yarn`, and running Prettier as you usually would, there's no additional configuration to worry about!
+
+There are three official plugins under development:
+
+* [`@prettier/plugin-python`](https://github.com/prettier/prettier-python)
+* [`@prettier/plugin-php`](https://github.com/prettier/prettier-php)
+* [`@prettier/plugin-swift`](https://github.com/prettier/prettier-swift)
+
+All three of these plugins are still under active development and aren't _quite_ really to throw production code at, but keep an eye on them as they will be soon! If you're interested in helping build those plugins, check out their repository issues list or run it against your own code and start raising bugs! Similarly, if you're interested in building a new language in Prettier and need some help getting it off the ground, [create an issue in the prettier repo](https://github.com/prettier/prettier/issues/new) and let us help you out.
```

Comment:
`prettier` -> `Prettier`

Reply:
I meant to use lower case here to denote the repository.